std/stdio.d

```diff
@@ -1522,6 +1524,143 @@ for every line.
     }
 
 /**
+Creates a file with a randomly generated name and returns an open File to it.
+
+The name will start with the given prefix and be located in the given directory.
+If no directory is given, then it will default to the result of
+$(XREF file, tempDir).
+
+There is nothing special about the file created other than the fact that its
+name is randomly generated, and it defaults to being in the temp directory on
+the system. It will only be deleted if explicitly deleted or if it's in the temp
+directory, and the temp is cleared out (as happens on system startup or shutdown
+on some systems).
+
+Params:
+    prefix = Optional prefix for file name.
+    suffix = Optional suffix for file name (which also provides a way to give
+             the file an extension).
+    dir    = Directory of temporary file.
```

Comment:
Ah, that's true. Well, originally, I had it is `dir = std.file.tempDir()`, but importing std.file in std.stdio at the module level or at the struct level caused a static assertion to be hit as explained where `tempDir` is called in the code. So, I went with treating the empty string as the default parameter and put the std.file import at local scope. And so, unless we just overload the function so that the directory isn't really an optional parameter but a different overload, we're kind of stuck. But maybe I should overload it so that `""` then works if explicitly provided for `dir`.